### PR TITLE
Make runtime logging note stand out more

### PR DESCRIPTION
**Pull request by keyur32**
moved it up to troubleshooting section as it's a common issue that people seem to be hitting this week

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-3/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810241517408153-513%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=M2XO0g5DpCwEhhMEgjDQ%2ByXqz1OIRo5zMP2oL2h1aRg%3D&st=2018-10-24T15%3A13%3A59Z&se=2018-11-24T15%3A18%3A59Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/excel/custom-functions-best-practices.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-3/docs/excel/custom-functions-best-practices.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C10%5C24%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201810241631263113-513%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=M4zIIdyc42Nx%2FhUBlDtdNl4RLUynxVuIXPGbM%2BIgcVo%3D&st=2018-10-24T16%3A27%3A42Z&se=2018-11-24T16%3A32%3A42Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
@Rick-Kirkham Do you mind looking this over? I just want to make sure the way we are giving instructions for giving feedback (i.e. step 1 > step 2, using the >s to separate steps) is correct. Thanks!

File: docs/excel/custom-functions-best-practices.md [hunk 1]
```diff
@@ -30,6 +30,12 @@ function getComment(x) {
 }
 ```
 
+## Troubleshooting
+
+If you are testing your add-in in Office on Windows, you should enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues.
+
+To report feedback to the Excel Custom Functions team about this method of troubleshooting, send the team feedback. To do this, select File > Feedback > Send a Frown. Sending a frown will provide the necessary logs to understand the issue you are hitting.
```
Comment (Rick-Kirkham) [inline docs/excel/custom-functions-best-practices.md]: Separate navigation steps with a pipe `"|"` symbol, not a `>`. Also, bold UI strings, so in this line "File" "Feedback" and "Send a Frown" should be bold.

Comment (mscharlock) [reply docs/excel/custom-functions-best-practices.md]: Super - will change! Thanks

**Pull request merged**